launcher_scripts/nemo_launcher/core/k8s_templates/rlhf_ppo/rlhf-ppo-actor.yaml

```diff
@@ -0,0 +1,75 @@
+{{ $config := .Values.trainingConfig }}
+
+apiVersion: kubeflow.org/v1
+kind: PyTorchJob
+metadata:
+  name: rlhf-ppo-actor
+  labels:
+    app: rlhf-ppo-actor
+spec:
+  pytorchReplicaSpecs:
+    Worker:
+      replicas: {{ .Values.actor.nodes }}
+      template:
+        spec:
+          containers:
+          - name: pytorch
+            image: {{ .Values.image.trainingImage }}
+            env:
+              {{- range $key, $value := $config.envVars }}
+              - name: {{ $key }}
+                value: {{ $value | quote }}
+              {{- end}}
+            {{ if eq $config.wandbKey "nil" }}
+            command: ["torchrun"]
+            args:
+              - "--nnodes={{ .Values.actor.nodes }}"
+              - "--rdzv-backend=c10d"
+              - "--rdzv-endpoint=rlhf-ppo-actor-worker-0"
+              - "--nproc_per_node={{ .Values.actor.numGPUs }}"
+              - "/opt/NeMo-Aligner/examples/nlp/gpt/train_gpt_ppo_actor.py"
+              - "--config-path=/config"
+              - "--config-name=config_actor.yaml"
+              - "remote_critic_rm.critic.ip=rlhf-ppo-critic-worker-0.{{ $config.namespace }}.svc.cluster.local"
+              - "remote_critic_rm.critic.port=5567"
+            {{ else }}
+            command: ["bash", "-c"]
+            args:
+              - "wandb login {{ $config.wandbKey }} && torchrun --nnodes={{ .Values.actor.nodes }} --rdzv-backend=c10d --rdzv-endpoint=rlhf-ppo-critic-0 --nproc_per_node={{ .Values.actor.numGPUs }} /opt/NeMo-Aligner/examples/nlp/gpt/train_gpt_ppo_actor.py --config-path=/config --config-name=config_actor.yaml remote_critic_rm.critic.ip=rlhf-ppo-critic-worker-0.{{ $config.namespace }}.svc.cluster.local remote_critic_rm.critic.port=5567"
```

Comment:
I think rdzv-endpoint will also need to change once you parametrize the pytorchjob name with the release name